packages/svelte/scripts/compile-test.js

```diff
@@ -1,10 +1,15 @@
 // Compile all Svelte files in a directory to JS and CSS files
 // Usage: node scripts/compile-test.js <directory>
 
-import { mkdirSync, readFileSync, writeFileSync } from 'fs';
-import path from 'path';
-import glob from 'tiny-glob/sync.js';
+import {
+	mkdirSync as mkdir_sync,
```

Comment:
we don't have a convention of aliasing external packages to `snake_case`

Reply:
Yes, I understand.
If svelte 5 accepts the eslint rule,
You can use the `eslint-disable-next-line id-match` comment here.

Reply:
I appreciate that this rule found a lot of places that should be updated. However, I think it's just going to end up being too annoying to do eslint disable wherever we're calling an external package and think we probably can't accept this PR as a result. I really appreciate the effort you put into it. I think we could accept the fixes you identified if they were split out separately from the lint rule itself - though it might be better to wait until Svelte 5 for that as we're going to be throwing away all the existing compiler code

Reply:
Thank you for the thorough review.  
The changes were intended for validating the effectiveness of the rule, and I believe applying them in the current version wouldn't bring significant benefits.